<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Up_load</class>
 <widget class="QWidget" name="Up_load">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="IP_Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port_Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Load_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Show_Edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>451</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MES回传信息：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="reply_Edit_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>165</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Tai Le</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MES回传计时：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="reply_Edit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
